src/Bundle/test/config/sylius/resources.yaml

```diff
@@ -3,6 +3,11 @@ sylius_resource:
         locale_provider: test.translation_locale_provider
 
     resources:
+        app.board_game:
+            driver: false
```

Comment:
Why don't we use `null` here?

Reply:
null is for default value and this will be configured with doctrine/orm value instead.

Reply:
But if we have a default value if null will be provided, this false needs to be defined explicitly. And then, we trigger an error. I don't see the use case for this functionality. Can you write some examples of usage of it?

Reply:
It does not trigger an error
https://github.com/Sylius/SyliusResourceBundle/blob/1fcf7dcac28f26217736cc6febd28c550859d3d3/src/Bundle/DependencyInjection/SyliusResourceExtension.php#L145